src/main/java/ru/sbtqa/tag/pagefactory/stepdefs/GenericStepDefs.java

```diff
@@ -420,6 +421,18 @@ public void swipeToText(String direction, String text) throws SwipeException {
         MobileExtension.swipeToText(DirectionStrategy.valueOf(direction.toUpperCase()), text);
     }
 
+    /**
+     * Swipe until text is visible for Android
+     *
+     * @param strategy contains or exact
+     * @param text text on page to swipe to
+     * @throws SwipeException if the text is not found
+     */
+    @And("ru.sbtqa.tag.pagefactory.swipeToTextAndroid")
+    public void swipeToTextAndroid(String strategy, String text) throws SwipeException {
```

Comment:
^user swipe to text \"([^\"]*)\" using match strategy \"([^\"]*)\"$ - first text, second strategy
swipeToTextAndroid(String strategy, String text) - first strategy, second text

will this match properly?

Reply:
Of course, my mistake.
I'll fix it